### PR TITLE
add flash message if activation by administrator

### PR DESCRIPTION
**Pull request by mennax**
If User settings > Activation mode = Administrator
No message is displayed after user registration

### CODE REVIEW

File: components/Account.php [hunk 1]
```diff
@@ -325,6 +326,15 @@ public function onRegister()
                 Flash::success(Lang::get(/*An activation email has been sent to your email address.*/'rainlab.user::lang.account.activation_email_sent'));
             }
 
+            /*
+             * Activation is by the admin, show message
+             * For automatic email on account activation rainLab.Notify plugin is needed
+             */
+            if ($adminActivation) {
+
+                Flash::success(Lang::get(/*You have successfully registered. Your account is not yet active.*/'rainlab.user::lang.account.activation_by_admin'));
```
Comment (bennothommo) [inline components/Account.php]:
```suggestion
                Flash::success(Lang::get(/*You have successfully registered. Your account is not yet active and must be approved by an administrator.*/'rainlab.user::lang.account.activation_by_admin'));
```

File: lang/en/lang.php [hunk 2]
```diff
@@ -182,6 +182,7 @@
         'login_first' => 'You must be logged in first!',
         'already_active' => 'Your account is already activated!',
         'activation_email_sent' => 'An activation email has been sent to your email address.',
+        'activation_by_admin' => 'You have successfully registered. Your account is not yet active.',
```
Comment (bennothommo) [inline lang/en/lang.php]: @mennax Would you mind making the same change to your Italian translation update as well? 
```suggestion
        'activation_by_admin' => 'You have successfully registered. Your account is not yet active and must be approved by an administrator.',
```

**Comment by mennax** [comment]
Ok @bennothommo, Thanks for the review.
Sorry, I'm newbee with GitHub. I commited changes in my fork and now (surprisingly :) ) I see it above #0609c4d. Tha's all?

File: components/Account.php [hunk 3]
```diff
@@ -325,6 +326,15 @@ public function onRegister()
                 Flash::success(Lang::get(/*An activation email has been sent to your email address.*/'rainlab.user::lang.account.activation_email_sent'));
             }
 
+            /*
+             * Activation is by the admin, show message
+             * For automatic email on account activation rainLab.Notify plugin is needed
+             */
+            if ($adminActivation) {
+
```
Comment (LukeTowers) [inline components/Account.php]:
```suggestion
            if ($adminActivation) {
```

File: components/Account.php [hunk 4]
```diff
@@ -325,6 +326,15 @@ public function onRegister()
...
+                Flash::success(Lang::get(/*You have successfully registered. Your account is not yet active and must be approved by an administrator.*/'rainlab.user::lang.account.activation_by_admin'));
```
Comment (LukeTowers) [inline components/Account.php]:
```suggestion
            if ($adminActivation) {
                Flash::success(Lang::get(/*You have successfully registered. Your account is not yet active and must be approved by an administrator.*/'rainlab.user::lang.account.activation_by_admin'));
```

Comment (mennax) [reply components/Account.php]: ok

File: components/Account.php [hunk 5]
```diff
@@ -325,6 +326,14 @@ public function onRegister()
                 Flash::success(Lang::get(/*An activation email has been sent to your email address.*/'rainlab.user::lang.account.activation_email_sent'));
             }
 
+            /*
+             * Activation is by the admin, show message
+             * For automatic email on account activation rainLab.Notify plugin is needed
```
Comment (LukeTowers) [inline components/Account.php]:
```suggestion
             * For automatic email on account activation RainLab.Notify plugin is needed
```

**Pull request merged**